documentation/Get-PnPLargeListOperationStatus.md

```diff
@@ -0,0 +1,30 @@
+---
+Module Name: PnP.PowerShell
+title: Get-PnPLargeListOperationStatus
+schema: 2.0.0
+applicable: SharePoint Online
+external help file: PnP.PowerShell.dll-Help.xml
+online version: https://pnp.github.io/powershell/cmdlets/Get-PnPLargeListOperationStatus.html
+---
+ 
+# Get-PnPLargeListOperationStatus
+
+## SYNOPSIS
+Get the status of a large list operation. Currently supports Large List Removal Operation.
+
+## SYNTAX
+
+```powershell
+Get-PnPLargeListOperationStatus [-ListId] <ListId> [-OperationId] <OperationId> [-Connection <PnPConnection>]
+```
+
+## DESCRIPTION
+
+Allows to get the status of a large list operation.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```powershell
+Get-PnPLargeListOperationStatus -ListId 9ea5d197-2227-4156-9ae1-725d74dc029d -OperationId 924e6a34-5c90-4d0d-8083-2efc6d1cf481
+```
```

Comment:
Can you also please add the parameter description for `Identity` and `OperationId` parameters ? You can check other docs in the documentation for reference

Reply:
I've updated the documentation for the parameter description. Thanks!